extensions/amp-analytics/integrating-analytics.md

```diff
@@ -0,0 +1,58 @@
+# Integrate your analytics tools in AMP HTML
+
+## Overview
+
+If you operate a software-as-a-service tool for publishers to better understand their traffic and visitors, you may want to integrate your service into AMP-Analytics. This will enable your customers to view traffic patterns in AMP HTML.
+
+## Before you begin
+
+Before you can add your analytics service to AMP HTML runtime, you may need to:
+* Identify the kinds of [variables](analytics-vars.md) and [requests](amp-analytics.md#requests) you'll need in an AMP HTML document for your analytics service.
+* Identify the triggers that result in analytics requests being sent from a page that would be relevant for your service.
+* Consider if and how you will [track users across](cross-origin-tracking.md) first-party and third-party AMP contexts.
+* Determine how your analytics dashboard handles AMP traffic.
+* Identify any missing functionality in AMP-Analytics, and note them as requested features on the GitHub project.
```

Comment:
"and [file requests](https://github.com/ampproject/amphtml/issues/new) for needed features"